packages/toolpad-app/src/components/SplitPane.tsx

```diff
@@ -38,6 +38,9 @@ const WrappedSplitPane = React.forwardRef<
       // Some sensible defaults
       minSize={20}
       maxSize={-20}
+      style={{
+        position: 'relative',
```

Comment:
Yep, makes sense. Perhaps it could make a bit more sense on the `paneStyle` instead, since that's the exact rectangle we want that editor to size to?

Reply:
That was my initial though too, I tried it but it seems that `paneStyle` is responsible for other components so I had to use `style`

Reply:
Ok, so it's the whole splitpane sizing badly. I'm a bit concerned that this will mess up existing splitpane usage. Perhaps it's safer to [keep it absolute](https://github.com/mui/mui-toolpad/pull/678) and have it size against its parent?